.eslintrc

```diff
@@ -8,6 +8,7 @@
     "react/button-has-type": 0,
     "import/prefer-default-export": 0,
     "react/require-default-props": 0,
+    "prefer-promise-reject-errors": 0,
```

Comment:
why

Reply:
ANTD validation expects you to reject invalid objects as Promise.reject(string), doesn't make sense to wrap everything in error objects